apps/admin-x-activitypub/src/components/Activities.tsx

```diff
@@ -91,7 +91,8 @@ const Activities: React.FC<ActivitiesProps> = ({}) => {
         data,
         fetchNextPage,
         hasNextPage,
-        isFetchingNextPage
+        isFetchingNextPage,
+        isLoading // Add this to destructure isLoading from the hook
```

Comment:
```suggestion
        isLoading
```

Comment not needed 👍